docs/installation.mdx

```diff
@@ -123,4 +123,33 @@ Version:        1.0.0
 Go version:     go1.20.2
 Git commit:     b58e7b910ca556973d111e9bd734a71baef03db2
 Git tree state: clean
+```
+
+## To Validate the Checksum of ORAS CLI Binaries
+
+
+The following commands can be used to verify the ORAS CLI binaries using GPG:
+
+1. First, we import a GPG Key which can be used for verification:
+
+```
+$ curl -sSL https://github.com/qweeah.gpg | gpg --import -
+```
+
+2. You can run the following command to check if the key has been imported:
+
+```
+$ gpg --list-keys
+pub   rsa4096 2023-02-28 [SC] [expires: 2024-02-28]
+      BE6FA8DDA48D4C230091A0A9276D8A724CE1C704
+uid           [ unknown] Billy Zha <jinzha1@microsoft.com>
+```
+
+3. Verify our binaries using the command:
+
+```
+$ gpg --verify oras_1.0.0_linux_amd64.tar.gz.asc oras_1.0.0_linux_amd64.tar.gz
+gpg: Signature made Mon Mar 20 15:51:28 2023 IST
+gpg:                using RSA key BE6FA8DDA48D4C230091A0A9276D8A724CE1C704
+gpg: Good signature from "Billy Zha <jinzha1@microsoft.com>" [unknown]
```

Comment:
Does this only work if Billy created the release?

Reply:
I think we need to use the key which was used for the release. But I'm not very sure.

cc: @qweeah

Reply:
If we have to, we could tell them to trust all the keys

Reply:
> Does this only work if Billy created the release?

User can get the releaser info from `Notes` section of every release. The proper steps should be
1) Find a target release in https://github.com/oras-project/oras/releases
2) Get releaser's info from the `Notes` section in the release notes.
3) Import releaser's key
4) Download binary and signature(.asc)
5) gpg --verify